src/app/playground/actions/updateDevUserRole.ts

```diff
@@ -0,0 +1,25 @@
+"use server";
+import { Tables } from "@/lib/db";
+import { getSupabaseClient } from "@/utils/supabase/supabaseClient";
+import { UpdateDevUserRole } from "../components/UpdateDevUserForm";
```

Comment:
Variable name is odd. Can it be `UpdateUserPayload`?

Reply:
Sir, we can say payload , its type and using in multiple places in `devUserForm` and `actions`